apiserver/common/machine.go

```diff
@@ -125,12 +126,21 @@ func ModelMachineInfo(st ModelManagerBackend) (machineInfo []params.ModelMachine
 		instId, err := m.InstanceId()
 		switch {
 		case err == nil:
-			mInfo.InstanceId = string(instId)
+			mInfo.InstanceId = instId
 		case errors.IsNotProvisioned(err):
 			// ok, but no instance ID to get.
 		default:
 			return nil, errors.Trace(err)
 		}
+		displayName, err := m.DisplayName()
```

Comment:
No need for separate block of code here - just get the instance id and display name together above